packages/libtool.rb

```diff
@@ -8,22 +8,22 @@ class Libtool < Package
   source_sha256 'e3bd4d5d3d025a36c21dd6af7ea818a2afcd4dfc1ea5a17b39d7854bcd0c06e3'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-x86_64.tar.xz',
+#    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-armv7l.tar.xz',
+#     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-armv7l.tar.xz',
+#       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-i686.tar.xz',
+#     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-x86_64.tar.xz',
```

Comment:
Please remove the lines instead of commenting them.

Reply:
Corrected in commit 495c345